packages/react-core/src/components/NumberInput/examples/NumberInput.md

```diff
@@ -57,6 +57,49 @@ class BasicNumberInput extends React.Component {
 }
 ```
 
+### Without onChange
```

Comment:
@mcoker @mcarrano should this example be added to core.

Reply:
I think so, though I wonder if we want to use a `readonly` input. Typically readonly implies the user can't modify it, but they still can here with the plus/minus buttons and using keyboard up/down arrows. For example, you can't use `readonly` with `required` since it's assumed the user can't interact with it.

Reply:
I can remove, I just thought if we have the ability to modify with buttons only we should show it. lmk